src/views/viewer/VideoView/VideoView.tsx

```diff
@@ -23,6 +23,7 @@ import { formatVideoViewsAndDate } from '@/utils/video'
 import { ChannelLink, InfiniteVideoGrid } from '@/components'
 import { usePersonalData, useRouterQuery } from '@/hooks'
 import { useVideo, useAddVideoView } from '@/api/hooks'
+import knownLicenses from '@/mocking/data/raw/knownLicenses.json'
```

Comment:
If we use this data in a real component, even on production, that means it shouldn't be in `@/mocking` since it's not mocked in any way